vllm/entrypoints/openai/serving_chat.py

```diff
@@ -1208,15 +1208,8 @@ async def chat_completion_stream_generator(
                             # check to see if there's anything left to stream
                             remaining_call = expected_call.replace(actual_call, "", 1)
                             # set that as a delta message
-                            delta_message = DeltaMessage(
-                                tool_calls=[
-                                    DeltaToolCall(
-                                        index=index,
-                                        function=DeltaFunctionCall(
-                                            arguments=remaining_call
-                                        ).model_dump(exclude_none=True),
-                                    )
-                                ]
+                            delta_message = self._maybe_create_remaining_args_delta(
```

Comment:
Thanks~ @luke396  

I noticed that #31438 seems to be addressing the same issue.

What do you think about #31438?